app/controllers/settings/featured_tags_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Settings::FeaturedTagsController < Settings::BaseController
+  layout 'admin'
+
+  before_action :authenticate_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    @featured_tag   = FeaturedTag.new
+    @most_used_tags = Tag.most_used(current_account).where.not(id: @featured_tags.map(&:id)).limit(10)
+  end
+
+  def create
+    @featured_tag = FeaturedTag.new(featured_tag_params)
+    @featured_tag.account = current_account
```

Comment:
Not sure of current project practice, but you can also do this as `@featured_tag = current_account.featured_tags.new(featured_tag_params)`